lib/util.py

```diff
@@ -35,6 +35,7 @@ def watchStatusChanged(self):
     def onNotification(self, sender, method, data):
         if sender == 'script.plex' and method.endswith('RESTORE'):
             from windows import kodigui
+            getAdvancedSettings()
```

Comment:
Do we really need this? Isn't it already added globally with `advancedSettings = AdvancedSettings()` right after the class definition.

Reply:
Yes, but if the user exits p4k by minimizing it, then changes the addon settings and then reactivates p4k, the changes won't be reflected.

Reply:
Ah, yeah. Good call!